ui/shared/components/form/ReduxFormWrapper.jsx

```diff
@@ -268,7 +271,11 @@ class ReduxFormWrapper extends React.Component {
   }
 
   componentWillUpdate(nextProps) {
-    if (nextProps.submitSucceeded && nextProps.closeOnSuccess && !nextProps.noModal) {
+    if (nextProps.submitSucceeded && nextProps.closeOnSuccess && nextProps.noModal) {
+      this.props.closeInlineModal()
+      this.props.handleClose(true)
```

Comment:
Before, if the `noModal` prop was passed `this.props.handleClose(true)` was NOT called, so this changes the behavior of every other time this component is rendered with that prop, almost definitely in a breaking way. Additionally, those forms do not have `closeInlineModal` passed in so calling it as a function will also break them. Please update this so it works for your components with the new prop without breaking backwards compatibility